### PR TITLE
Issue 845 more annotated paragraph

### PR DESCRIPTION
**Pull request by moufort**
Issue #845 
I add valid label and a default label. If the label is not a valid label, then we use the default label

### CODE REVIEW

File: src/Microdown-HTMLExporter-Tests/MicHTMLVisitorTest.class.st [hunk 1]
```diff
@@ -147,25 +147,30 @@ MicHTMLVisitorTest >> testCreateAnchorWithLink [
...
-	
 	result := writer convertMicString: '# This is a title
-!!LabelOfAnnotatedBlock Body of annotated block
+!!note Body of annotated block
 
 this is another string'.
-	self assert: result trimBoth equals: '<h1>This is a title</h1>', newLine ,
-'<section id="LabelOfAnnotatedBlock"><label for="LabelOfAnnotatedBlock">Body of annotated block</label></section>', newLine ,
-'<p>this is another string</p>'.
+	self
```
Comment (Ducasse) [inline src/Microdown-HTMLExporter-Tests/MicHTMLVisitorTest.class.st:154]: To me it looks strange to create a heading 1 for a note. 
Why did you suggest this?

File: src/Microdown/MicAnnotatedParagraph.class.st [hunk 2]
```diff
@@ -26,9 +29,23 @@ MicAnnotatedParagraph >> accept: aVisitor [
...
+MicAnnotatedParagraph >> defaultLabel [ 
+	
+	^ #note
+]
+
+{ #category : 'testing' }
+MicAnnotatedParagraph >> initialize [
+
+	super initialize.
+	validLabel := { #note . #important . #todo }
```
Comment (Ducasse) [inline src/Microdown/MicAnnotatedParagraph.class.st]: Could use plurals eg validLabels 
Tx

File: src/Microdown-HTMLExporter-Tests/MicHTMLVisitorTest.class.st (hunk 1 above)
Comment (Ducasse) [reply src/Microdown-HTMLExporter-Tests/MicHTMLVisitorTest.class.st:154]: And we can let it for now and see later.

Comment (moufort) [reply src/Microdown-HTMLExporter-Tests/MicHTMLVisitorTest.class.st:154]: The test with h1 already exist when I have adding the verifiedLabels. I only correct the test for let him pass green

**Comment by Ducasse** [comment]
Thanks for the explanation.

**Pull request merged**